daemon/algod/api/algod.oas3.yml

```diff
@@ -1195,6 +1195,25 @@
         ],
         "type": "object"
       },
+      "ApplicationLocalReference": {
+        "description": "References an account's local state for an application.",
+        "properties": {
+          "account": {
+            "description": "Address of the account with the local state.",
+            "type": "string",
+            "x-algorand-format": "Address"
+          },
+          "app": {
+            "description": "Application ID of the local state application.",
```

Comment:
We should probably scour the whole oas2 file for uses of "App" vs "Application" in descriptions.  I don't care which, but I think I'm seeing different choices in different places.

Reply:
Application is much more common than app